examples/ant_world/ant_world.cc

```diff
@@ -41,6 +43,18 @@ inline second_t getCurrentTime()
 {
     return units::make_unit<second_t>(glfwGetTime());
 }
+
+std::unique_ptr<HID::JoystickBase<HID::JAxis, HID::JButton>> createJoystick(GLFWwindow *window)
+{
+    try
+    {
+        return std::make_unique<HID::Joystick>(0.25f);
+    }
+    catch(std::runtime_error &)
+    {
+        return std::make_unique<HID::JoystickGLFWKeyboard>(window);
```

Comment:
Might be worth printing a little message here, in case the user has plugged in a joystick which just isn't working.